styles/RichText.module.scss

```diff
@@ -0,0 +1,83 @@
+.bold {
+    font-weight: $weight--bold;
+}
+
+.italic {
+    font-style: italic;
+}
+
+.paragraphLight {
+    color: var(--color--grey);
```

Comment:
Line-heights aren't responsive here - refactor the SCSS to include line heights that vary as the font size changes? @include font-size(s) should also include line height?